src/__tests__/userGroupChangemakerPermissions.int.test.ts

```diff
@@ -0,0 +1,316 @@
+import request from 'supertest';
+import { app } from '../app';
+import {
+	db,
+	createChangemaker,
+	createOrUpdateUserGroupChangemakerPermission,
+	loadUserGroupChangemakerPermission,
+	removeUserGroupChangemakerPermission,
+} from '../database';
+import { expectTimestamp, loadTestUser } from '../test/utils';
+import {
+	mockJwt as authHeader,
+	mockJwtWithAdminRole as authHeaderWithAdminRole,
+} from '../test/mockJwt';
+import { KeycloakId, keycloakIdToString, Permission } from '../types';
+import { NotFoundError } from '../errors';
+
+const mockKeycloakOrganizationId = '123e4567-e89b-12d3-a456-426614174000';
+
+describe('/userGroups/changemakers/:changemakerId/permissions/:permission', () => {
+	describe('PUT /', () => {
+		it('returns 401 if the request lacks authentication', async () => {
+			const organizationAsChangemaker = await createChangemaker(db, null, {
+				taxId: '11-1111111',
+				name: 'Example Inc.',
+				keycloakOrganizationId: mockKeycloakOrganizationId,
+			});
+			const changemaker = await createChangemaker(db, null, {
+				taxId: '11-1111112',
+				name: 'Example Inc.',
+				keycloakOrganizationId: null,
+			});
+			await request(app)
+				.put(
+					`/userGroups/${keycloakIdToString(organizationAsChangemaker.keycloakOrganizationId as KeycloakId)}/changemakers/${changemaker.id}/permissions/${Permission.MANAGE}`,
+				)
+				.send({})
```

Comment:
I apparently missed this when reviewing before. I don't think we should be required to send an empty body. @slifty what do you think?